tests/unittests/test_cli.py

```diff
@@ -218,18 +219,27 @@ def test_wb_devel_schema_subcommand_parser(self):
             self.stderr.getvalue())
 
     def test_wb_devel_schema_subcommand_doc_content(self):
-        """Validate that doc content is sane from known examples."""
+        """Validate that doc content is sane from known examples.
+
+        Ensure that schema doc is returned
+        """
+
+        # Note: patchStdoutAndStderr() is convenient for reducing boilerplate,
+        # but inspecting the code for debugging is not ideal
+        # contextlib.redirect_stdout() provides similar behavior as a context
+        # manager
         stdout = io.StringIO()
-        self.patchStdoutAndStderr(stdout=stdout)
-        self._call_main(['cloud-init', 'devel', 'schema', '--docs', 'all'])
-        expected_doc_sections = [
-            '**Supported distros:** all',
-            ('**Supported distros:** almalinux, alpine, centos, cloudlinux, '
-             'debian, eurolinux, fedora, openEuler, opensuse, photon, rhel, '
-             'rocky, sles, ubuntu, virtuozzo'),
-            '**Config schema**:\n    **resize_rootfs:** (true/false/noblock)',
-            '**Examples**::\n\n    runcmd:\n        - [ ls, -l, / ]\n'
-        ]
+        with contextlib.redirect_stdout(stdout):
+            self._call_main(['cloud-init', 'devel', 'schema', '--docs', 'all'])
+            expected_doc_sections = [
+                '**Supported distros:** all',
+                ('**Supported distros:** almalinux, alpine, centos, '
+                    'cloudlinux, debian, eurolinux, fedora, openEuler, '
```

Comment:
This line may to cause problems openEuler vs. openeuler with #1117 fix I think.

Reply:
I don't think that issue in the PR is related to this. This test just validates that the supported distros in the `cc_ntp` meta config end up in the final doc. currently that list comes from here:

cloud-init/cloudinit/config/cc_ntp.py
```
distros = ['almalinux', 'alpine', 'centos', 'cloudlinux', 'debian',             
            'eurolinux', 'fedora', 'openEuler', 'opensuse', 'photon',            
            'rhel', 'rocky', 'sles', 'ubuntu', 'virtuozzo']  
```

so `system_info()` is not involved